receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -215,3 +220,20 @@ func buildGaugeOrCounterPoint(parsedMetric *statsDMetric, now *timestamppb.Times
 	metricTypeSetter(parsedMetric, isDouble)
 	return point, nil
 }
+
+func buildTimerPoint(parsedMetric *statsDMetric, now *timestamppb.Timestamp, metricTypeSetter func(parsedMetric *statsDMetric)) (*metricspb.Point, error) {
+	var point *metricspb.Point
+	parsedMetric.unit = "ms"
+	f, err := strconv.ParseFloat(parsedMetric.value, 64)
+	if err != nil {
+		return nil, fmt.Errorf("timer: parse metric value string: %s", parsedMetric.value)
```

Comment:
If I came across this message, I would find it cryptic. Perhaps add what kind of parsing failed?

```suggestion
		return nil, fmt.Errorf("timer: failed to parse metric value to float: %s", parsedMetric.value)
```

Or perhaps just wrap the underlying error?

```suggestion
		return nil, fmt.Errorf("timer: failed to parse metric value to float: %w", err)
```

Reply:
Thanks, make sense. I will revise to 
`return nil, fmt.Errorf("timer: failed to parse metric value to float: %s", parsedMetric.value)`